### PR TITLE
update version of integration services

### PR DESCRIPTION
**Pull request by ivancorrales**
Updated Integration services  in this PR make use of the v3.6.7 Infrastructure SDK

### CODE REVIEW

File: bundle.yml [hunk 1]
```diff
@@ -16,50 +16,50 @@ integrations:
...
   - name: nri-rabbitmq
-    version: 2.2.4
+    version: 2.3.0
   - name: nri-redis
-    version: 1.6.1
+    version: 1.6.2
   - name: nri-snmp
-    version: 1.2.0
+    version: 1.3.0
     archs: [ amd64 ]
```
Comment (roobre) [inline bundle.yml:60]: Is `nri-snmp` still `amd64` only? If it's not, I'd remove the `archs` line to make it available in all of them.

File: bundle.yml [hunk 2]
```diff
@@ -16,50 +16,50 @@ integrations:
...
+    version: 4.3.5
   - name: nri-f5
-    version: 2.2.1
+    version: 2.3.0
   - name: nri-haproxy
     version: 2.2.1
   - name: nri-jmx
-    version: 2.4.5
+    version: 2.4.6
     archs: [ amd64 ]
```
[the remark this answers is not in the document]
Comment (ivancorrales) [reply bundle.yml]: Done

File: bundle.yml (hunk 1 above)
Comment (ivancorrales) [reply bundle.yml]: Done

**Comment by roobre** [comment]
CI is failing with `2021/05/12 12:38:54 error fetching integrations: got status 404 when fetching https://download.newrelic.com/infrastructure_agent/binaries/linux/arm/nri-snmp_linux_1.3.0_arm.tar.gz`

it seems that snmp is not yet in the prod repo. I'll ping @newrelic/caos abiut this to see if they are aware.

**Review by roobre — APPROVED** [review]
Looking great, thanks!

**Comment by ivancorrales** [comment]
Updated Integration SDK in the services

**Pull request merged**